README.md

```diff
@@ -1,27 +1,60 @@
 go-skeleton
 -----------
-A skeleton for go microservices
+This project is used as boilerplate example for the structure of a go microservice. A template to get you started so to speak.
+
+A skeleton for go microservices was the core idea, so that's where the name comes from.
 
 
 ## Motivation
 Besides avoiding to copy it from the last microservice I've written, a friend asked how I'd do it.
 That's is how I'm doing it right now
 
 
+## Download
+
+Clone the repository using 
+
+### SSH
+
+```bash
+git clone git@github.com:AndersonQ/go-skeleton.git
+```
+
+### https
+
+```bash
+git clone https://github.com/AndersonQ/go-skeleton.git
+```
+or using 
+
+```bash
+go get -u github.com/AndersonQ/go-skeleton.git
```

Comment:
👍 
you need to remove the `.git` at the end
```suggestion
go get -u github.com/AndersonQ/go-skeleton
```

also you could add `go install github.com/AndersonQ/go-skeleton` as well